src/formula.rs

```diff
@@ -134,7 +134,7 @@ pub const PATHFINDING_VNPC_REACHABILITY_LIMIT: i32 = 500;
 pub const PLAYER_PATH_WALKING_DELAY: Duration = Duration::from_millis(300);
 
 pub fn exploration_radius(mental_state: Mind) -> i32 {
-    use crate::player::Mind::*;
+    use crate::player::Mind::{High, Sober, Withdrawal};
```

Comment:
I don't want to merge any of these explicit enum variant imports.

Reply:
I will revert it.